src/uwuw/uwuw_preprocessor.cpp

```diff
@@ -96,11 +96,10 @@ pyne::Material uwuw_preprocessor::create_new_material(pyne::Material material, s
 
   std::string material_name;
   // make a new name
+  material_name = "mat:" + material.metadata["name"].asString();
```

Comment:
We can npw combine this line with the declaration of `std::string material_name` above.